app/models/provider.rb

```diff
@@ -912,6 +947,11 @@ def set_defaults
       self.billing_information = {} if billing_information.blank?
       self.consortium_id = nil unless member_type == "consortium_organization"
       self.non_profit_status = "non-profit" if non_profit_status.blank?
+      if member_type === "consortium_organization"
```

Comment:
Is this supposed to be a `==` or are you explicitly checking to see if it is included in a list?

https://dev.to/baweaver/understanding-ruby-triple-equals-2p9c

Reply:
Good catch.  Will fix that.